packages/@okta/vuepress-site/docs/guides/revoke-tokens/removeusersess/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Remove a User session
+---
+
+Separate from access and refresh tokens, there is also the Okta session cookie that provides access to your Okta organization and applications. For a more complete explanation of Okta User sessions, see the [Sessions API documentation](/docs/reference/api/sessions/). You can revoke Okta sessions in one of two ways:
```

Comment:
In the previous page, the referencing phrase used when linking to an API document was "Okta OIDC & OAuth 2.0 reference"  and here we have "Sessions API documentation"; for consistency, could we use
**OIDC & OAuth 2.0 API reference**
**Sessions API reference**